src/modules/app/StoreActionType.js

```diff
@@ -51,7 +48,11 @@ export type FetchCategoryActionType = {|
   |}
 |}
 export type FetchCategoryFailedActionType = {|
-  type: 'FETCH_CATEGORY_FAILED', message: string
+  type: 'FETCH_CATEGORY_FAILED',
+  params: {|
```

Comment:
I'm not sure here whether we should use `params`. The flux spec says to use payload:
https://github.com/redux-utilities/flux-standard-action

I think we can just add them directly to the root of the action.

Reply:
AFAIK we use params nearly all the time. So I changed this for the sake of consistency. If we don't want this, we should do this in a task imo.

Reply:
If it is more consistent now then this is good :+1: